3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/EnrichmentConfigsSpec.scala

```diff
@@ -63,7 +63,26 @@ class EnrichmentConfigsSpec extends Specification with ValidationMatchers with D
       val schemaKey = SchemaKey("com.snowplowanalytics.snowplow", "anon_ip", "jsonschema", "1-0-0")
```

Comment:
We'll need a new schema for this in `iglu-central`.

Reply:
https://github.com/snowplow/iglu-central/issues/1008